zulipterminal/config/symbols.py

```diff
@@ -6,7 +6,7 @@
 MESSAGE_CONTENT_MARKER = "▒"  # Options are '█', '▓', '▒', '░'
 QUOTED_TEXT_MARKER = "░"
 MESSAGE_HEADER_DIVIDER = "━"
-CHECK_MARK = "✓"
+CHECK_MARK = "✔"
```

Comment:
This seems like a separate change, particularly given it's part of the API. This might simplify writing the tests, depending if we only use these constants.